DOC-1875: RC - Billing alert user limit (#2346)
RedisLabs/redislabs-docs: content/rc/security/access-management.md

| **Email** | The address used for alerts and other email messages regarding the account | 
| **Alert emails** | Enable to be notified when subscription databases cross certain thresholds, such as exceeding memory limits or latency requirements |
| **Operational emails** | Notifications about subscription and database changes, such as creating or deleting a database |

> | **Billing emails** | Notifications when bills are issued, paid, and so on.<br/>_Note:_ Limited to a single user. |

Reviewer: ```suggestion
| **Billing emails** | Notifications when bills are issued or paid.<br/>_Note:_ Limited to a single user. |
```

Author: Fixed
